CONTRIBUTING.md

```diff
@@ -0,0 +1,149 @@
+   - Name the branch according to our [branching guide](#branching-guide).
+   - Branches are useful since they isolate your changes from incoming changes from upstream. They also enable you to create multiple PRs from the same fork.
+4. Make and commit your changes to your branch.
+   - You can run static code checks b running `npm run lint`.
+   - You can run tests locally by running `npm run test:local:node`.
+   - Please follow our [Commit Messages](#commit-messages) guidance.
+5. Add new tests corresponding to your change, if applicable.
+6. Build the repository with your changes.
+   - Make sure that the builds are clean.
+   - Make sure that the tests are all passing, including your new tests.
+7. Make sure your branch has all changes from the main/release branch of the source repository.
+8. Increment the npm package version in the package.json (adhere to [semantic versioning](https://semver.org/)). Run `npm i` to make sure the change is propagated to package-lock.json.
+9. Create a pull request (PR) against the repository's **main** branch or the **release/\*** branch of the release you want to contribute to.
+   - State in the description what issue or improvement your change is addressing.
+   - Check if all the Continuous Integration checks are passing.
+10. Wait for feedback or approval of your changes from the team.
+11. When the PR is approved, and all checks are green, your PR will be merged.
+    - This will automatically trigger our continuous deployment pipeline, which builds and publishes a new version.
+    - You can delete the branch you used for making the change.
+
+### Branching Guide
+
+Branches must match the following pattern:
+
+```text
+<prefix>/[<issue-id>]<description>
+```
+
+- `<prefix>`: must be one of `chore`, `feature` or `bugfix`
+- `issue-id`: id of the GitHub issue created in step 1
+- `description`: concise description of the changes
+
+### Commit Messages
+
+Please format commit messages as follows:
+
+```text
+<prefix>: Summarize change in 65 characters or less
+
+Provide more detail after the first line. Leave one blank line below the
+summary and wrap all lines at 72 characters or less.
+
+If the change fixes an issue, leave another blank line after the final
+paragraph and indicate which issue is fixed in the specific format
+below.
+
+Fix #42
+```
+
+- `<prefix>`: must be one of `chore`, `feat`, `fix`, `refactor`, `test`
+
+Also do your best to factor commits appropriately, not too large with unrelated things in the same commit, and not too small with the same small change applied N times in N different commits.
+
+### PR - CI Process
+
+The Enmeshed continuous integration (CI) system will automatically perform the required builds and run tests (static code checks as well as unit/integration tests) for PRs. Every step of the CI process needs to succeed before the PR can be merged.
```

Comment:
irgendwie klingt `The Enmeshed continuous integration (CI) system` ein bisschen hoch gestochen für ein paar yml files in jedem Repo